composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "illuminate/collections": "^10.0",
-        "illuminate/support": "^10.1"
+        "illuminate/support": "^10.3"
```

Comment:
Why the version bump?

Reply:
i dont even remember changing this it mustve been when I ran a command at some point